src/widgets/mentors/ui/mentors.module.scss

```
@@ -0,0 +1,44 @@
+.mentors-container {
+  background: $color-gray-100;
+}
+
+.mentors-content {
+  flex-direction: row;
+
+  @include media-tablet-large {
+    flex-direction: column;
+    gap: 40px;
+  }
+}
+
+.mentors-info {
+  display: flex;
+  flex-direction: column;
+
+  .paragraph {
+    max-width: 640px;
+
+    @include media-tablet-large {
+      max-width: 100%;
+    }
+  }
```

we can simplify this

```suggestion
  max-width: 640px;

  @include media-tablet-large {
    width: 100%;
  }
```

---

Done